src/Sylius/Bundle/AdminBundle/Resources/views/Product/Show/_header.html.twig

```diff
@@ -11,10 +11,24 @@
     </div>
     <div class="middle aligned column">
         <div class="ui right floated buttons">
-            <a class="ui labeled icon button {% if not product.enabled %} disabled {% endif %}" href="{{ path('sylius_shop_product_show', {'slug': product.slug, '_locale': product.translation.locale}) }}">
-                <i class="angle right icon"></i>
+            <div class="ui floating dropdown labeled icon button">
+                <i class="share alternate icon"></i>
+                <span class="text">
                 {{ 'sylius.ui.show_product_in_shop_page'|trans }}
-            </a>
+                </span>
+                <div class="menu">
+                    <div class="scrolling menu">
+                        {% for channel in product.channels %}
+                            {% set url = channel.hostname is not null ? 'http://' ~ channel.hostname ~ path('sylius_shop_product_show', {'slug': product.slug, '_locale': channel.defaultLocale.code}) : url('sylius_shop_product_show', {'slug': product.slug, '_locale': channel.defaultLocale.code}) %}
```

Comment:
I suppose we should not hardcode `http://` there (a website can be accessible from https, for example)

Reply:
Have you any idea how to do it? If i delete 'http://'  i get error, I don't have better resolve but i suspect that hardcoding is not good way.